ipapython/ipaldap.py

```diff
@@ -1656,3 +1656,27 @@ def entry_exists(self, dn):
             return False
         else:
             return True
+
+
+class LDAPConnectionManager(object):
```

Comment:
Because this is related only to ldapi, I would rename it to LDAPI connection manager